teacher.rb

```diff
@@ -0,0 +1,50 @@
+class Teacher
+  attr_reader :age, :phase, :target_raise, :salary
+  attr_accessor :name
```

Comment:
Creating a 'person' class from which both students and teachers can inherit makes sense to me.  It also seems like a case could be made for extracting some of the behaviors to a module(modules?)...for example, the phase and performance setter methods; is it the responsibility of a teacher to set their own phase, etc?